examples/ssh/sshcert/sshcert_test.go

```diff
@@ -84,30 +87,38 @@ func TestCASignerCreation(t *testing.T) {
 }
 
 func TestSshCertCreation(t *testing.T) {
-	caSigner, err := newSshSignerFromPem(caSecretKey)
-	if err != nil {
-		t.Error(err)
-	}
-
-	principals := []string{"guest", "dev"}
-
 	alg := jwa.ES256
-
 	signingKey, err := util.GenKeyPair(alg)
 	if err != nil {
 		t.Fatal(err)
 	}
-	email := "arthur.aardvark@example.com"
-	pkt, err := mocks.GenerateMockPKTokenWithEmail(signingKey, alg, email)
-	if err != nil {
-		t.Fatal(err)
+
+	// extra ID token payload claims
+	mockEmail := "arthur.aardvark@example.com"
+	extraClaims := map[string]any{
+		"email": mockEmail,
 	}
 
+	op, err := clientmocks.NewMockOpenIdProvider(t, extraClaims)
```

Comment:
Nice! I thought about using this pattern when I added email, but decided to do it in a later PR that I never got around to. Glad to see this get added.

You moved this is to clientmocks (client.mocks) to remove some of the cyclical dependency issues with client?

Reply:
I just put it there because I started with the client code. I'm still moving it around to see where it feels best as I modify other tests.